packages/queries/src/getSelectableColumns.ts

```diff
@@ -0,0 +1,41 @@
+import { ColumnSchema, DbSchema, TableSchema } from './types/DbSchema';
```

Comment:
Can you move this file to `src/schema`. We should only keep the most important files at the root.